woocommerce/woocommerce-gutenberg-products-block
# Critical flows: Shopper → Cart/Checkout → Can use express checkout

This is an E2E test to check if the express payment button is available on cart and checkout blocks


Fixes #5783

### Testing

#### Automated Tests

### Manual Testing

How to test the changes in this Pull Request:

1. If e2e tests pass in CI on this branch, should be good 👍 
2. As an extra test you can run `npm run test:e2e --  cart-checkout-payments` to run this test only.
